clients/admin-ui/src/features/integrations/add-integration/allIntegrationTypes.tsx

```diff
@@ -15,6 +15,8 @@ import {
   ConnectionType,
 } from "~/types/api";
 
+import MICROSOFT_SQL_SERVER_TYPE_INFO from "../integration-type-info/microsoftSQLServerInfo";
```

Comment:
I'm not super familiar with typescript, is there any way we can make this an absolute path instead of a relative one?